client/src/PublishTenderForm.js

```diff
@@ -111,7 +114,7 @@ const PublishTenderForm = () => {
 
 				if (!response.ok) {
 					const errorData = await response.json();
-					throw new Error(errorData.error || "Failed to publish tender.");
+					throw new Error(errorData.code || "Failed to publish tender.");
```

Comment:
We can revisit this when we start working on the UI, but the code is not to be shown to the user, it's so the UI knows which message to show